_posts/2022-06-28-trino-summit-call-for-speakers.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Announcing the 2022 Trino Summit"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+Hello again Trino nation,
+
+We are pleased to announce the 2022 Trino Summit, which will be happening in a
```

Comment:
We are pleased to announce the upcoming 2022 Trino Summit. The event is scheduled as hybrid event on the 10th of November 2022. You can therefore join us online, or you can make the trip to San Francisco in meet us in the Commonwealth Club at the waterfront in downtown.


Something like that